File: mycore-mets/src/main/ts/model/simple/MCRMetsAltoLink.ts
/*
 * This file is part of ***  M y C o R e  ***
 * See http://www.mycore.de/ for details.
 *
 * MyCoRe is free software: you can redistribute it and/or modify
 * it under the terms of the GNU General Public License as published by
 * the Free Software Foundation, either version 3 of the License, or
 * (at your option) any later version.
 *
 * MyCoRe is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 * GNU General Public License for more details.
 *
 * You should have received a copy of the GNU General Public License
 * along with MyCoRe.  If not, see <http://www.gnu.org/licenses/>.
 */

///<reference path="MCRMetsFile.ts"/>
namespace org.mycore.mets.model.simple {
    export class MCRMetsAltoLink {
        constructor(public altoFile: MCRMetsFile, public begin: string, public end: string) {
        }

        public static copy(file: MCRMetsAltoLink) {
            return new MCRMetsAltoLink(file.altoFile, file.begin, file.end);
        }
    }
}

File: mycore-mets/src/main/java/org/mycore/mets/model/simple/MCRMetsAltoLink.java
package org.mycore.mets.model.simple;

public class MCRMetsAltoLink {

    public MCRMetsAltoLink(MCRMetsFile file, String begin, String end) {
        this.file = file;
        this.begin = begin;
        this.end = end;
    }

    private MCRMetsFile file;

    private String begin;

    private String end;

    public MCRMetsFile getFile() {
        return file;
    }

    public void setFile(MCRMetsFile file) {
        this.file = file;
    }

    public String getBegin() {
        return begin;
    }

    public void setBegin(String begin) {
        this.begin = begin;
    }

    public String getEnd() {
        return end;
    }

    public void setEnd(String end) {
        this.end = end;
    }
}
